internal/querycoordv2/observers/collection_observer_test.go

```diff
@@ -232,18 +256,23 @@ func (suite *CollectionObserverSuite) isCollectionTimeout(collection int64) bool
 	replicas := suite.meta.ReplicaManager.GetByCollection(collection)
 	channels := suite.targetMgr.GetDmChannelsByCollection(collection, meta.CurrentTarget)
 	segments := suite.targetMgr.GetHistoricalSegmentsByCollection(collection, meta.CurrentTarget)
-
 	return !(exist ||
 		len(replicas) > 0 ||
 		len(channels) > 0 ||
 		len(segments) > 0)
 }
 
+func (suite *CollectionObserverSuite) isCollectionLoadedContinue(collection int64, beforeTime time.Time) bool {
+	fmt.Println(suite.meta.GetCollection(collection).UpdatedAt, beforeTime, "aaaaaaaaa")
+	return suite.meta.GetCollection(collection).UpdatedAt != beforeTime
```

Comment:
suite.meta.GetCollection(collection).UpdatedAt.After(beforeTime) should be better